src/main/java/org/zoo/EarthCreatures.java

```diff
@@ -1,6 +1,4 @@
-package org.example;
-import java.util.ArrayList;
-import java.util.SortedMap;
+package org.zoo;
 
 public abstract class  EarthCreatures {
```

Comment:
Why is this class abstract? I don't see any abstract methods in it

Reply:
make two methods abstract, leave as abstract